src/FSharp.AWS.DynamoDB/Script.fsx

```diff
@@ -21,13 +21,13 @@ let account = AWSCredentialsProfile.LoadFrom("default").Credentials
 let ddb = new AmazonDynamoDBClient(account, RegionEndpoint.EUCentral1) :> IAmazonDynamoDB
```

Comment:
Pretty sure this credential approach is long gone - I used

```
open Amazon.Runtime.CredentialManagement
let (success, creds) = CredentialProfileStoreChain().TryGetAWSCredentials("default")
let ddb = if success then new AmazonDynamoDBClient(creds) :> IAmazonDynamoDB else failwith "Unable to load default credentials"
```

Reply:
worked for me too, will delete old code